README.md

```diff
@@ -1214,7 +1214,7 @@ cooper-mbp:black cooper$ ~/venvs/b/bin/black-primer
 
 Failed projects:
 
-## flake8-bugbear:
+### flake8-bugbear:
```

Comment:
This now does not match output :D

Reply:
Yeah, that was causing issue with the old build logic... thankfully #1456 solves that :)

Reply:
All good - I didn’t really care and would even add a ‘#’ into the output to make things easier. I just wanted something in the outputto skip projects easily in vim / less etc.

Reply:
![image](https://user-images.githubusercontent.com/63936253/82738575-736b5500-9d06-11ea-92c2-89e93ba92e7e.png)

:D